csharp/src/Microsoft.ML.OnnxRuntime/OrtValue.shared.cs

```diff
@@ -261,6 +261,23 @@ public ReadOnlySpan<byte> GetStringElementAsSpan(int index)
             }
         }
 
+        /// <summary>
+        /// This API resizes String Tensor element to the requested amount of bytes (UTF-8)
+        /// and returns a span to the resized buffer. This API is useful when you have UTF-8 data
+        /// that you want to quickly stick into the OrtValue native memory.
+        /// </summary>
+        /// <param name="bytesCount">requested buffer size in bytes</param>
+        /// <param name="index">flat index of the element in the string tensor</param>
+        /// <returns></returns>
+        public Span<byte> GetMutableStringElementAsSpan(int bytesCount, int index)
```

Comment:
I wonder if we should instead have another `FillStringTensorElement` overload which takes utf-8 bytes?

Reply:
Fixed.